include/blob/blob_client.h

```diff
@@ -53,6 +53,15 @@ namespace azure { namespace storage_lite {
             m_client = std::make_shared<CurlEasyClient>(max_concurrency, ca_path);
         }
 
+        blob_client(std::shared_ptr<storage_account> account,
+                    std::shared_ptr<executor_context> context,
+                    std::shared_ptr<CurlEasyClient> client)
```

Comment:
>std::shared_ptr<CurlEasyClient> client [](start = 20, length = 38)

`client` should be internal logic that is not exposed to customers. Please make the 3rd argument `int max_concurrency` instead.

Reply:
I let the ctor with client for future tests and to join all the ctor logics in to one flow, however it is private so the customer will not use it.
Please recheck and share your thoughts.